website/src/react-native/css/react-native.css

```diff
@@ -1013,10 +1013,24 @@ h2 {
   text-rendering: optimizelegibility; }
 
 .docs-prevnext {
-  min-width: 320px;
   max-width: 640px;
   margin: 40px auto;
-  padding-bottom: 20px; }
+  padding-bottom: 20px;
+}
```

Comment:
From looking through the rest of this file it looks like we prefer having the ending curly on the previous line